app/main.py

```
@@ -36,89 +124,80 @@ def process_csv(file_content: bytes, filename: str) -> None:
-    Uploads a CSV file and processes it in the background.
-    
-    Args:
-        file (UploadFile): The uploaded CSV file.
-
-    Raises:
-        HTTPException: If the file type is invalid.
-
-    Returns:
-        BaseResponse: A response indicating that the CSV file is being processed.
-    """
+async def upload_csv(file: UploadFile = File(...)) -> BaseResponse:
```

Ruff

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)